src/proteomes/components/entry/Overview.tsx

```diff
@@ -0,0 +1,130 @@
+                </>
+              ),
+            },
+            renderColumnAsInfoListItem(ProteomesColumn.proteinCount),
+            {
+              title: 'Gene Count',
+              content: (
+                <>
+                  {`${data.geneCount}`}
+                  {
+                    // data?.geneCount > 0 && // TODO: API always returns zero (Leo investigating) so removing the condition until this fixed to at least show the link for UX purposes
+                    data.superkingdom && data.taxonomy.taxonId && (
+                      <>
+                        {' '}
+                        <a
+                          href={ftpUrls.referenceProteomes(
+                            data.id,
+                            data.superkingdom,
+                            data.taxonomy.taxonId
+                          )}
+                        >
+                          Download one protein sequence per gene (FASTA)
+                        </a>
+                      </>
+                    )
+                  }
+                </>
+              ),
+            },
+            {
+              title: 'Proteome ID',
+              content: data.id,
+            },
+            (data.taxonomy.taxonId || data.taxonomy.scientificName) && {
+              title: 'Taxonomy',
+              content: (
+                <Link
+                  to={{
+                    pathname: generatePath(
+                      LocationToPath[Location.TaxonomyEntry],
+                      {
+                        accession: data.taxonomy.taxonId.toString(),
+                      }
+                    ),
+                  }}
+                >
+                  {[data.taxonomy.taxonId, data.taxonomy.scientificName]
+                    .filter(Boolean)
+                    .join(' · ')}
+                </Link>
+              ),
+            },
+            {
+              title: 'Last modified',
+              content: <time>{data.modified}</time>,
+            },
+            data.genomeAssembly?.assemblyId && {
+              title: 'Genome assembly and annotation',
+              content: (
+                <GenomeAssemblyAndAnnotationView {...data.genomeAssembly} />
+              ),
+            },
+            data.genomeAssembly?.level &&
+              renderColumnAsInfoListItem(ProteomesColumn.genomeRepresentation),
+            renderColumnAsInfoListItem(ProteomesColumn.cpd),
+            data.proteomeCompletenessReport?.buscoReport && {
+              title: <BuscoAbbr />,
+              content: (
+                <div className="busco">
+                  <div className="busco__legend">
+                    <BuscoLegend />
+                  </div>
+                  <BuscoView
+                    report={data.proteomeCompletenessReport?.buscoReport}
+                  />
+                </div>
+              ),
+            },
+          ].filter(Boolean) as InfoData
+        }
+      />
```

Comment:
There should also be a description as part of the overview for some proteomes. Is that missing from the data payload?

Reply:
Good catch. This was just an oversight on my part - updated now